pages/halloffame.md

```diff
@@ -16,6 +16,12 @@ This list includes successful pull requests, issues and other interactions with
 - Project: [Jenkins](https://github.com/jenkins-infra/jenkins.io)
 - [Pull Request](https://github.com/jenkins-infra/jenkins.io/pull/1233)  
   - [Approved Changes](https://jenkins.io/doc/book/system-administration/security/)
+  
+###  Team Mavericks
+- Members: Niluja Biswal,, Neeharica Kodali, Durgabhavani Peta, Kasiviswanath Reddy Poreddy
+- Project: [TeamMates] (https://github.com/TEAMMATES/teammates)
+- [Open Security Issues] (https://github.com/TEAMMATES/teammates/issues/8183) (https://github.com/TEAMMATES/teammates/issues/8178)
```

Comment:
Please fix the last link. In markdown, each link will require its own anchor text. Like this: 
- Open Security Issues:  
[Issue 1](https://github.com/TEAMMATES/teammates/issues/8183)   
[Issue 2](https://github.com/TEAMMATES/teammates/issues/8178)